product-matrix.json

```diff
@@ -5,18 +5,20 @@
       "version": "3.1",
       "ideaProduct": "android-studio",
       "ideaVersion": "173.4670197",
-      "dartPluginVersion": "173.4700",
+      "dartPluginVersion": "181.4203.498",
       "sinceBuild": "173.0",
-      "untilBuild": "173.*"
+      "untilBuild": "173.*",
+      "filesToSkip": ["src/io/flutter/editor/FlutterCompletionContributor.java"]
     },
     {
       "name": "IntelliJ",
       "version": "3.2",
       "ideaProduct": "android-studio",
-      "ideaVersion": "173.4688006",
-      "dartPluginVersion": "173.4700",
-      "sinceBuild": "173.0",
-      "untilBuild": "173.*"
+      "ideaVersion": "181.4705630",
+      "dartPluginVersion": "181.4203.498",
+      "sinceBuild": "181.0",
+      "untilBuild": "181.*",
+      "filesToSkip": ["src/io/flutter/editor/FlutterCompletionContributor.java"]
     },
     {
       "name": "IntelliJ",
```

Comment:
We should probably remove this third entry, otherwise we'll be building two plugins that declare compatibility with 2018.1.